src/j1cli.js

```diff
@@ -447,6 +449,15 @@ async function mutateQuestions(j1Client, questions, operation) {
       )}.`
     );
   }
+  if (newFile) {
+    var jsonString = JSON.stringify(questions, null, 2);
+
+    const writeFile = util.promisify(fs.writeFile);
```

Comment:
Because this creates a function that should need any kind of state, it's often advantageous to signal this by putting this just after the imports (~ line 10 or so). It doesn't change any functionality but declutters this function for an improved reading experience later. :smile:

Reply:
Did you want to move this call or intentionally leave it here?

Reply:
I figured it was fine here since its only being used in this function